scene/gui/rich_text_label.cpp

```diff
@@ -3611,6 +3611,72 @@ bool RichTextLabel::is_selection_enabled() const {
 	return selection.enabled;
 }
 
+int RichTextLabel::get_selection_from() const {
+	if (!selection.active || !selection.enabled) {
+		return -1;
+	}
+
```

Comment:
At this point you can simply do this;
`return selection.from_frame->lines[selection.from_line].char_offset + selection.from_char;`

Reply:
Thanks! It works much better using this.